src/quo2/screens/messages/system_message.cljs

```diff
@@ -0,0 +1,67 @@
j6F4S0f8M52J+hdvgvqelZFGJBBLd/mWQGOLuLoT0dRXucrTBfvRt34mNf+jrWv+8WAqIwHzZPPE9m8Dlm78A7EE6exVWdwM+ejKHIUvnkCVrN4CwoWGJPEMbchI7+DqH4CLI4DZ4aWo2DT68lnKMQfTPQy5wAG6YSXvUDApl9GaqaCvqqqXTQuuPCzdXLGSFRwjrcZb+o3YJxd2Di4FbMxecxlO3AA4f60RUrY/2GbmxpH8NkPofKKw9gZrgdvVttlWgmZtZg10c/i/k1qzD/+D72AB7qBDbWqvV416f/GIEUQ8ZzVFNSIkt0zwO9aNcCuP7WCI5T4KtXO5gt59CSbJNNXriqp4jxxgQ2bX4FtyTy6EjVMXJ4DXIj3bw2XTvRgaFn29C5klWppar2BWksgwEhPZgLgmyJZbZgxqU197LGzw/cd6G8r1GArJln/dvarvYPVftrqYgRx0tPsK+vuujKhHHoZRPPHezBrbesQ9j6KXp2AcOzaQwd7EbHxk8wNYRgXfs+ZtwATj30Q1SjCbznX+8hJU0vEjICzV2eBeL6dMzHfGQX5sbOYHiOLbKfQZGUWF9nGe959ziZpACu2TYqnT+7uZDaRtvH+M+NT3PaxJlD1sL4qQYOEml+NOwgnOOQhb1KYwHxOQs7drRaI6eFgjdfTNaLKqDzNmQr/5S6I5Twv1mYTCO0poAgJz7DozaGz1axel0/63AWDw9R+xttPHpgNVJsRjb23Ejgw5m97FnZegOJyjTCbFg0ZmKD8FgREbLjUzo5WzZEuNi4dSec0Uc4Oa4iauXxL3cUCIZIp74aQ3jNOFAUmpwNTIgWTszDnwW2f3ASRx/vxIpO5peZAEsrUV6CkHpKV5spVbPGEmgLH+jJbjH3jsBFdoi9JgcsXZGb5u7Sapm7g+kJjJ1NscRavJGP9++ysWP7OpwePcEJawAnXiYSzMveP/J5vesIxS212yNKGmzljmvRvnGTIi092aay0GMIFycgZ+SMjh1b15CmnueANICBLTVE+qqIbS4jtKsIaw2/Hycj3BLhCI1j80O0mOwNYknd/v5n0Z5kKDkpzFY5b9hEoQlzPWGNmHztAqdNrmIQ7gg8dfSu15PzDUnRlX+4f89dP0je/+JYHLIbua2FKG+qFZvWryHQcpAIM8nYVYQJQrTULvL+YWVp1XIyHnXXVXv+grKXR7pEvSm8wAwZcJj0gOl8A6vXzOGWd3NMlowhN9qC6bM9qNWosBrRHL3aM+fUH5doPUSK00XhvOBzwrT5vc9ivBJQwKfsh9RxTeYAbIUtwTG1xv3h/Uf3vpGMBi6xnj1+dl86dfXHW6JGRybtYfPWG2HlzsApTLBBjKBKljbREcPZ0grsvJaTYC2wQET4isAUbK4tUFhNbkGu6nM0Nofy1K+woiWJjYl9zPI9+OWhOIeohNuNMNviVsyc6sLMi0Xkz/SyXZ5AMFmDnyKqO83KIaVmPokVDL1njwfRSyyyYrxE2KwrTsD1/CORQy994FLyXVIBsoZPv/ydOz+X6a/Nzm5fsZ4g5qqv0/oV3PD+o1jXOY8NbSV0dG1Ca/915O0MtTdPSEdtCWcvmFzCQMha2bejFbMYH36KJGqYHdoJCkb294YqyaQ6wrECrdjAXJZ8YCSNdMcoXni6n9iiDivJ8sm5QuUY2ePp9WjpyePg8zF0s1vsOO6oexDr3x8/+uIlhb9sBci6/5HT+75x23qtf3VioCW6H0cO92PN1WcRIBAy8xxvr/gUrMxWtUlJ9uWoi5vNFlxb2DwppGbDs9XeftSGcX3/fUyoTyIRSCB1dhjxq1JItp3g5EfHy79cjfYdRcJmHzOTvWSKSyy5pNVPdqJrLaGtvQ6VacY5q1SV5VZ2hfTL1riCeXf0yJF/c7lyXbYCZH3roZHBr902nZ8bC1/X018KRSPM2GYQlTyTWuJj5MQ3KvbV9Zr7dUyCkLoaVhiokZMrz07DH9uP6aPfwcaNd8JMFOkRsmWFFUI6uDw/Z9MTzEyjdcNZnHi2CxFS5HF2n7oT5WAjhYeJFssjJtrUZJjVpFgiW6dhPGvlOkvT/zHz5PG9VyKTiStcsZtfuWv+4Y59muY9wTv3+QEBOzG2ynnEVhKA6I7aHCF/IEVjo1qiO88/hJmhacLfl9Aam+LEt4+MDwGKWydbQ6Q5y1khGR0taRFxGipEYtF5dPXPcEaYwNipVnrGLJ46ncTEfBrDDP+rO/NklV3el/U/mB5sePptXQ9ms7jCdcUKkJX+oPpz1v7Ct0K7gxvNr9pnAn0geAHHYGF/lBl4mG7v4NQzz7J3OMZGiFOdBjvK0BRGx4O0GmF1NoXW7im2uEySqxm3Yw3YIwlYKYIcjtS9Shl9Vx3D0499AJH2Kex7KoPJuQq6SbYeO92CP7i5TsrNyFXmw3ds/9pTd+FNrisKgQvXnQ85hz988zt+0nVVVMu0D20MV78XCln3IVz7Pq39IIb3N5BK+Xj5GQtr+ivs7T1Mz7ehZ8MIzpzuxMo12eYeHoJ1P86yN84yOc/qwXmAzg5OZ6dZmA3ha3/fitZoA5WGwDqSnFYlR97zr3b93tFPZT49OYjfYr2ZvSEXXf6hUJ89Zw2YaxNfxfxUn81yPTXfj8MvdKI10Y72zgmMjPQyWWWRSbjIjsTxrvc+z0FqXub0HGUxZEif2QeYDzbJRKcdtcl+sjo2vv1gCmNTDjlEPxcncfN3D+XuyuYO5/AWrLdMAUtX9TcYCJT03fZK66Z7//qGvmtunsIIG6oVHHBGUpM4fHA91qw9Q9zv4yMfeKa5w17xdUEUj6VRmGhDdFUC8c4sSVUre+joqp/893+I7/vlc/sG8RavN5UDLrXC13PWAGFd6/jm7va+Myent09OZbb3bXllW716dV/numOpuWKk7/FDbcwVW7B+BZGiG+WEysmFYna2Ml4fWfnhA4enX90+2PkJwfAn8Lta/wxj9JtBu4B9/QAAAABJRU5ErkJggg=="}
```

Comment:
Maybe separating these huge strings into constants? So the object itself gets more readable. Same for images below. Not a big deal since it is a preview though.

Reply:
can't me move this into another file too?

Reply:
Done